snippets/fsharp/VS_Snippets_CLR_System/system.console.class/fs/example3.fs

```diff
@@ -0,0 +1,169 @@
+﻿
+// dump a range of Unicode characters as a 16x16 array
+// <Snippet4>
+open System
+open System.IO
+open System.Globalization
+open System.Text
+
+type uint = uint32
+
+let roundDownToMultipleOf (b:uint) (u:uint) : uint =
+    u - (u % b)
+
+let roundUpToMultipleOf (b:uint) (u:uint) : uint =
+    roundDownToMultipleOf b u |> (+) b
+
+let displayRange (start:uint) (``end``:uint) : unit =
+
+    let mutable upperRange : uint = 0x10FFFFu
+    let mutable surrogateStart : uint = 0xD800u
+    let mutable surrogateEnd : uint = 0xDFFFu
```

Comment:
These type annotations aren't required.

Reply:
Good point! Since the literal suffix already specifies the type explicitly.